vitrina/admin.py

```diff
@@ -0,0 +1,136 @@
+import json
+from typing import Any, Dict, Optional
+
+from django.contrib import admin
+from django.contrib.admin.models import LogEntry
+from django.http import HttpRequest
+from django.urls import reverse, NoReverseMatch
+from django.utils.html import format_html, escape
+import reversion
+from reversion.models import Revision, Version
+from reversion.admin import VersionAdmin
+
+from vitrina.utils import RevisionComment, RevisionSource
+
+
+class VersionInline(admin.TabularInline):
+    model = Version
+    extra = 0
+    can_delete = False
+
+    fields = (
+        "version_data",
+        "col_content_type",
+        "col_object_id",
+        "col_object_repr",
+    )
+    readonly_fields = fields
+
+    def has_add_permission(self, request: HttpRequest, obj: Optional[Revision] = None) -> bool:
```

Comment:
Let's use built-in python types and not from Typing


```suggestion
    def has_add_permission(self, request: HttpRequest, obj: Revision | None = None) -> bool:
```

Same for other methods as well

Reply:
https://github.com/atviriduomenys/katalogas/pull/2179/commits/447fd4af8020e9143139e7a5ee3e7526561d595e